examples/toolchains/cc/WORKSPACE

```diff
@@ -0,0 +1,33 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "rules_cc",
+    sha256 = "4dccbfd22c0def164c8f47458bd50e0c7148f3d92002cdb459c2a96a68498241",
+    urls = [
+        "https://github.com/bazelbuild/rules_cc/releases/download/0.0.1/rules_cc-0.0.1.tar.gz",
+    ]
+)
+
+load("@rules_cc//cc:repositories.bzl", "rules_cc_dependencies", "rules_cc_toolchains")
+
+rules_cc_dependencies()
+
+rules_cc_toolchains()
+
+local_repository(
+    name = "io_tweag_rules_nixpkgs",
+    path = "../../../"
+)
+
+load("@io_tweag_rules_nixpkgs//nixpkgs:repositories.bzl", "rules_nixpkgs_dependencies")
+rules_nixpkgs_dependencies()
+
+load("@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl",
+     "nixpkgs_git_repository",
+     "nixpkgs_cc_configure"
+)
+nixpkgs_git_repository(
+    name = "nixpkgs",
+    revision = "21.11",
```

Comment:
Please also set the `sha256` attribute so that this can be cached in Bazel's repository cache. It's not enabled in this example setup, but it makes for a better example - in the sense of following best practices.